docs/20_tutorial.md

```diff
@@ -14,10 +15,161 @@ the computers read those ballots, to determine whether the computers
+they will be encountered, with brief, clear instructions describing what
+is expected of the user(s) at each step, starting with the login screen
+and proceeding through the completion of the audit, or determination that
+a full hand count is required. First SOS, then County, then Public.
+
+## For County Administrators
+
+For the 2017 Colorado implementation, the full functionality of the RLA Tool requires exports from the 
+Dominion System ???. 
+***Colorado2017
+***which Dominion system?
+
+### Logging In
+
+Screenshot (County Admin Authentication) ---
+
+
+### <a name="comparison-audit-upload"></a>Comparison Audit Upload
+
+To prepare for upload the 
+ballot manifest and the CVR file must be first verified and hashed. 
+The RLA Tool does not provide a utility for the 
+verification and hashing. 
+***ballot manifest upload prep
+
+Once the ballot manifest and CVR files are verified and hashed, 
+they can be uploaded.
+
+Screenshot (page:county) ---
+
+Note that the RLA Tool does not support export of the
+tabulation results to the Election Night Reporting System. 
+
+### Ballot Polling Audit Upload
+The RLA Tool is designed for comparison audits, which require a voting
+system capable of exporting CVR files. However, Counties whose voting systems do not
+export CVR files can use the RLA Tool to upload ballot manifests, described
+in the [Comparison Audit Upload](#comparison-audit-upload) 
+section.
+
+### List of Ballot for Audit
+To see the list of ballots to be audited in the next round:
+Screenshot (???) ---
+
+
+
+## For County Audit Boards
+========
+
+### Signing In
+
+The Audit Board does not log directly into the RLA Tool with
+computer system usernames and passwords. However, whenever 
+the Audit Board begins to interact with the RLA Tool, either 
+at the beginning of an audit round or after taking a break, 
+the Audit Board must sign in.
+
+Screenshot (page:Acme County) ---
+
+Click on "Start Audit"
+
+Screenshot (page:Audit Board Sign In) ---
+
+Click "Next"
+
+### Auditing Ballots
+
+Note: do *not* use your browser's "back" arrow during the audit process.
+
+Screenshot (Ballot Verification) ---
+
+Before recording voter intent, please double-check that the paper ballot
+ID and Ballot Style match the ID and ballot style listed on the screen.
+Then record on the screen all final voter intent marked on the paper ballot.
+In case of an overvote, mark each of the (too many) choices the voter 
+intended; the RLA tool will recognize the overvote. In case the Audit 
+Board cannot reach consensus, mark the "No Consensus" box on the screen.
+
+Click "Review"
+
+Screenshot (Review Screen) ---
+
+Check that the information on the review screen reflects the 
+Audit Board's interpretation of the ballot. Note that in case of 
+an overvote, the review screen will say, simply, "overvote".
+
+Note: do *not* use your browser's "back" arrow during the audit process.
+If the review screen does not match the Audit Board's interpretation,
+click "Back".
```

Comment:
Agreed. Per my comment above, we can change this.

The reason it is like this right now is because the "wizard" is implemented as a component that changes at one particular URL path, `/county/audit`, with the current wizard view/"stage" determined by an internal variable. We can change this to make it so the active wizard stage match a further path part, e.g. `/county/audit/review`, or similar.